diff --git a/node_modules/@mdx-js/mdx/mdx-hast-to-jsx.js b/node_modules/@mdx-js/mdx/mdx-hast-to-jsx.js
index 0c0dbc3..a8e450e 100644
--- a/node_modules/@mdx-js/mdx/mdx-hast-to-jsx.js
+++ b/node_modules/@mdx-js/mdx/mdx-hast-to-jsx.js
@@ -111,6 +111,7 @@ function toJSX(node, parentNode = {}, options = {}) {
     <MDXLayout
       {...layoutProps}
       {...props}
+      // Let's see if this ever gets triggered
       components={components}>
 ${jsxNodes.map(childNode => toJSX(childNode, node)).join('')}
     </MDXLayout>
