diff --git a/node_modules/next/dist/server/next-server.js b/node_modules/next/dist/server/next-server.js
index 584edde..c9c3bbc 100644
--- a/node_modules/next/dist/server/next-server.js
+++ b/node_modules/next/dist/server/next-server.js
@@ -171,6 +171,7 @@ class Server {
         console.error(err);
     }
     async handleRequest(req, res, parsedUrl) {
+        console.log("server", req.url);
         var ref23, ref1, ref2, ref3, ref4, ref5;
         const urlParts = (req.url || '').split('?');
         const urlNoQuery = urlParts[0];
